Compiler/src/implementation/Scanner.cpp

```diff
@@ -257,7 +257,7 @@ Token Scanner::consumeGenericWordToken()
 	{
 		if (FilePosition + 1 == FileContents.size())
 		{
-			//Last character in file. Update pointer, and set End State To Valid.
+			//Last character in file. Update accumulator, and set End State To Valid.
```

Comment:
we're updating the pointer here. not the accumulator. Might consider changing wording in Cleanup PR but this fine for now